<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inference_panel</class>
 <widget class="QDialog" name="inference_panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MIVision Performance</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(226, 226, 226);</string>
  </property>
  <widget class="QLabel" name="logo_label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/amd-logo-150x150.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="NGPU_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>GPU:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Model_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
  </widget>
  <widget class="QLabel" name="modelName_label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="GPU_lcdNumber">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>64</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="digitCount">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="StartTime_label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>580</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logo_label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>150</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/vega_icon_150.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="elapsedTime_label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>650</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="header_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 48pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Performance</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>530</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="close_pushButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1710</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>440</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>440</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>440</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_0">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="digitCount">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="digitCount">
    <number>1</number>
   </property>
   <property name="intValue" stdset="0">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>460</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="digitCount">
    <number>1</number>
   </property>
   <property name="intValue" stdset="0">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="digitCount">
    <number>1</number>
   </property>
   <property name="intValue" stdset="0">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="GPUName_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
